pallets/core_mods/src/attest.rs

```diff
@@ -25,7 +25,13 @@ pub trait Trait: system::Config + did::Trait {
 pub struct Attestation {
     #[codec(compact)]
     priority: u64,
-    iri: Option<Iri>,
+    pub iri: Option<Iri>,
+}
+impl Attestation {
+    /// Create new Attestation
+    pub fn new(priority: u64, iri: Option<Iri>) -> Self {
+        Attestation { priority, iri }
+    }
```

Comment:
These changes are made in order to be able to create an attestation and access the size fo the IRI in the pallet and tests of fiat_filter

Reply:
@lovesh I think you mean `#[cfg(test)]`.

Reply:
Yes, thanks.

Reply:
AFAIK `cargo test` only applies `cfg(test)` to the crate you run it in, not to any dependencies.
Applying `cfg(test)` gives me the following error when running the fiat_filter pallet's tests:
```
error[E0599]: no function or associated item named `new` found for struct `core_mods::attest::Attestation` in the current scope
   --> pallets/fiat_filter/src/tests.rs:334:32
    |
334 |         let att = Attestation::new(1, None);
    |                                ^^^ function or associated item not found in `core_mods::attest::Attestation`
```

Reply:
There is a possible workaround, adding a feature to the pallet called `tests`. 
But removing cfg(test) seems like the more simple solution.
What do you think ?

Reply:
`struct Attestation` is purely for holding data and maintains no invariants. Both its fields should be public.

Reply:
> `struct Attestation` is purely for holding data and maintains no invariants. Both its fields should be public.

👍 That would remove the need for the `impl` entirely